### ISSUE TITLE
Remove rules for diffs that are no longer used/needed

### ISSUE THREAD

**Issue by gouttegd** [obophenotype/cell-ontology#2641]
The custom Makefile contains several rules to generate custom diffs:

(A) A diff between the local `-edit` file and the version from the master branch on GitHub. Exists in two versions: one with the imports removed (`reports/diff_edit_noimports.txt`), one with the imports merged (`reports/diff_edit_imports.txt`).

(B) A `obo-diff.txt` (unclear why it is called like that, it is generated from OWL/RDF files), produced by comparing the latest local build with the latest release, after removing (1) all non-CL stuff (thereby producing something akin to a “base” file) and (2) all annotations except `rdfs:label` (so that we only really compare logical axioms, plus labels).

(C) the “release-base-diff”, a standard ROBOT diff made by comparing the latest base release with the locally produced base.

(D) the “content summary”, also made by comparing the latest base release with the locally produced base, but produced by a custom Python script _and_ OAK’s diff.

**(C)** and **(D)** are produced and published upon every release. However, **(A)** and **(B)** are never automatically produced and only generated if someone explicitly invokes the corresponding rules. I think it is doubtful that **(A)** and **(B)** are needed at all (especially **(B)**, which is made quite redundant by **(C)** and **(D)**), and I’d be inclined to remove the corresponding rules entirely.

**Comment by matentzn** [comment]
I agree - if they are not invoked by GHActions we can remove them - if they are needed we can add a better version back in the future. I doubt anyone is really using it?

**Comment by gouttegd** [comment]
Leaving them for now as there is always a chance that an editor is generating those diffs locally (especially the diffs against the master branch) to test the impact of their changes before even submitting a PR.

But if nobody shows up in, say, one or two months to say that they use those rules, then I’ll submit a PR to remove them.

**Comment by gouttegd** [comment]
> But if nobody shows up in, say, one or two months to say that they use those rules

Nobody showed up in four months, so the rules are going away.
